keyboards/kyria/keymaps/tessachka/keymap.c

```diff
@@ -0,0 +1,241 @@
+/* Copyright 2019 Thomas Baart <thomas@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+enum layers {
+    QWERTY = 0,
+    LOWER,
+    RAISE,
+    ADJUST
+};
+
+enum custom_keycodes {
+    RGBRST = SAFE_RANGE,
+    KC_RACL
+};
```

Comment:
I don't actually see where any of the logic is for these keycodes.

Reply:
Good point, I'll ask Tessa what she had in mind with those keycodes.

Reply:
I am sorry to admit I have absolutely no idea what these do or what their purpose is.. I'm a complete code noob. I'm attempting to learn, but all I've done so far is copy other's sets of code and change just the keycodes I needed.. this was in the copied original map I used as a base for my crkbd map a short while ago apologies!! I have no idea what I'm doing when it comes to code AT ALL, lol. splitkb was super kind to build a Kyria code base for me to be able to mess with JUST the keycodes, which is all I know, but I gave him "my" crkbd map to base that on  <3

Reply:
Well, they're for custom keycodes, and used by `process_record_user`, as well as the keymap. 
https://docs.qmk.fm/#/feature_macros

Since, that function isn't being used right now, the enum for the keycodes isn't actually needed.  However, once it does get used, then that's when you'd need this enum.

Reply:
Usually when I see an `RGBRST` keycode, the logic is to set the RGB to static and red color (I think that's what the Crkbd does with it).

I only see one instance of `KC_RACL` in the repository, which is Right Alt when held and Colon when tapped.

https://github.com/qmk/qmk_firmware/blob/9ea9806d67dd3dfd775cf854b9becd35f3cdc748/keyboards/crkbd/keymaps/soundmonster/keymap.c#L353-L363

Reply:
Oooh, I see! I do want to be able to add macros in the future. Thank you so much for explaining!

Reply:
@noroadsleft I think for now, it should be okay to leave the enum in there. 

Ideally, I think it should be removed.  But for simplicity's sake here, it can be left. 

Also, I'll probably be helping tess out a lot with the keymap, and git/github in the near future, so I can address any issues then. 

Sound good?

Reply:
@drashna Sounds good.